.github/workflows/cross_version_compat.yml

```diff
@@ -0,0 +1,189 @@
+name: Cluster-Client version compatibility 
+
+on:
+  push:
+    branches: [master]
+  workflow_dispatch:
+jobs:
+  cross_compat:
+    timeout-minutes: 20
+    name: cluster(${{ matrix.cluster_type }}):${{ matrix.cluster_version }} x CLI:${{ matrix.cli_version }} 
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        rust: [stable]
+        cluster_type: [local] # fyi: k8 tests are unstable in github-runner
+        # Strategy: latest, stable, stable-1 release
+        cli_version: [latest, stable, 0.7.3]
+        cluster_version: [latest, stable, 0.7.3]
+        exclude:
+          - cli_version: latest
+            cluster_version: stable
+          - cli_version: latest
+            cluster_version: 0.7.3 
+          - cli_version: stable
+            cluster_version: 0.7.3 
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: infinyon/fluvio 
+      - run: helm version
+      - name: Install Rust ${{ matrix.rust }}
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          profile: minimal
+          override: true
+      - name: Install Minikube for Github runner 
+        if: matrix.os == 'ubuntu-latest' 
+        uses: manusa/actions-setup-minikube@v2.3.1
+        with:
+          minikube version: 'v1.16.0'
+          kubernetes version: 'v1.19.2'
+
+      - name: Open tunnel for Github runner
+        if: matrix.os != 'infinyon-ubuntu-bionic'
+        run: |
+          nohup  minikube tunnel --alsologtostderr > /tmp/tunnel.out 2> /tmp/tunnel.out &
+
+      - name: Setup Minikube for Linux
+        if: startsWith(matrix.os, 'infinyon-ubuntu')
+        run: |
+          pkill -f "minikube tunnel" || true
+          minikube delete
+          minikube start --driver=docker --kubernetes-version 1.19.6
+          nohup  minikube tunnel --alsologtostderr > /tmp/tunnel.out 2> /tmp/tunnel.out &
+      - name: Test minikube
+        run: |
+          minikube profile list
+          minikube status
+
+      - name: Install Fluvio for cluster @ (version ${{ matrix.cluster_version }}), 
+        if: matrix.cluster_version != 'stable'
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 5
+          max_attempts: 3
+          command: |
+            curl -fsS https://packages.fluvio.io/v1/install.sh | VERSION=${{ matrix.cluster_version }} bash
+            ~/.fluvio/bin/fluvio version
+
+      - name: Install Fluvio for cluster @ (version stable), 
+        if: matrix.cluster_version == 'stable'
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 5
+          max_attempts: 3
+          command: |
+            curl -fsS https://packages.fluvio.io/v1/install.sh | bash
```

Comment:
I feel like this is a workaround for a limitation in the `install.sh` script. I feel like a better approach would be to make `install.sh` recognize `VERSION=stable` properly so that this workflow can use the same scripting and just substitute different values as needed.

Reply:
Yep, this is a workaround. Modifying the installation script seems ok. I'll open an issue, assuming we don't want the default behavior shouldn't change. `VERSION=stable` will have the same behavior as not setting `VERSION` at all.